### DISCUSSION TITLE
OIDCPKCEMethod doesn't appear to respect the 'none' option in latest release

### DISCUSSION THREAD

**Discussion by eoliphan** [OpenIDC/mod_auth_openidc#1256] [General]
Hi, i'm using 2.4.16.1 with KeyCloak/RHSSO.   Keycloak at the moment explicitly doens't support PKCE for confidential clients.  I tried to fix this by setting '**OIDCPKCEMethod none**' but the redirect is still sending the **code_challenge**, etc vars and the **code_challenge_method** is set to S256

**Comment by zandbelt** [comment]
that appears to be a bug due to the config rewrite: it is fixed in 6ae48480b5a0646cb10f3e8aa63f3841fc2e94ae and is part of the new [2.4.16.2](https://github.com/OpenIDC/mod_auth_openidc/releases/tag/v2.4.16.2) release
